tests/test_type_constraints/test_unify.py

```diff
@@ -190,3 +190,29 @@ def test_diff_callable():
 
     unify_helper(c1, c2, TypeFail(
         f'Incompatible Types {c1} and {c2}'))
+
+
+# Polymorphic types
+def test_simple_polymorphic_call():
+    tc.reset()
+    tv1 = tc.fresh_tvar()
+    tv2 = tc.fresh_tvar()
+    fn1 = Callable[[tv1, tv2], bool]
```

Comment:
This test is okay, but use the helper `create_Callable`. There's something about setting `polymorphic_tvars` which will be important later.